src/search.py

```diff
@@ -24,11 +24,11 @@
 '''
 @search_bp.route('/search/', methods=['GET'])
 def search():
-    results = scraper.driver(request.args.get('name'), request.args.get('currency'),
+    results = scraper.driver(request.args.get('title'), request.args.get('currency'),
                              int(request.args.get('numProducts')))
 
     # if not descending, it is ascending
     results = result_formatter.sortList(results, request.args.get('sortBy'),
-        request.args.get('displayOrder').lower() == 'desc')
+        True if request.args.get('displayOrder').lower() == 'desc' else False )
```

Comment:
replace this with just `request.args.get('displayOrder').lower() == 'desc'`. The True if x else False is unnecessary and pylint complains about it

Reply:
True, easy fix, definitely overlooked that one